test/test_helper.exs

```diff
@@ -1,2 +1 @@
 ExUnit.start()
-Ecto.Adapters.SQL.Sandbox.mode(MindTheGapp.Repo, :manual)
```

Comment:
Is there a reason this was removed? IIRC this is needed to get proper database support in test.

Reply:
FYI, it looks like this didn't make it back in? That probably means that the tests are continuously adding data without it getting cleaned up.